### PR TITLE
[FIX] Fixes bug #77

### PR DESCRIPTION
**Pull request by hurrinico**
In order to solve https://github.com/OCA/stock-logistics-workflow/issues/77 i create this PR

### CODE REVIEW

File: stock_picking_deliver_uos/test/stock_partial_picking.yml [hunk 1]
```diff
@@ -3,13 +3,34 @@
...
-      - product_id: product.product_product_24
-        product_qty: 4
-        product_uos_qty: 5
-      - product_id: product.product_product_20
-        product_qty: 4
-        product_uos_qty: 5
+-
+  I add lines
+-
+  !python {model: stock.picking}: |
```
Comment (lepistone) [inline stock_picking_deliver_uos/test/stock_partial_picking.yml:9]: you can pass the `id` like you did in the `!record` block above, and you get `self` as a new-api recordset. Normally a good idea.

**Comment by gurneyalex** [comment]
even rebased, this is still travis :red_circle:

**Comment by Cedric-Pigeon** [comment]
@hurrinico Can we close this PR ? do you plan to achieve it ?

**Comment by pedrobaeza** [comment]
Closing as there's no news.

**Pull request closed without merging**